.github/ISSUE_TEMPLATE/fah-client-bug.md

```diff
@@ -0,0 +1,22 @@
+---
+name: FAHClient bug report
+about: If a bug in the Folding@Home client doesn't fit another category, click here.
+title: ''
+labels: FAHClient, defect
```

Comment:
It wasn't clear to me whether the `FAHClient` label should be used for this category or not, but I guessed that it made sense, as any bug that isn't in one of the other subcomponents is most likely in `FAHClient`.